packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -9,158 +8,156 @@ import {
 import { SystemColors } from '@microsoft/fast-web-utilities';
 import {
   accentFillRest,
+  baseHeightMultiplier,
   bodyFont,
   controlCornerRadius,
+  density,
   designUnit,
-  disabledOpacity,
+  fillColor,
   foregroundOnAccentRest,
-  neutralFillRest,
+  neutralForegroundHint,
   neutralForegroundRest,
+  strokeWidth,
   typeRampBaseFontSize,
   typeRampBaseLineHeight,
-  typeRampPlus3FontSize,
-  typeRampPlus3LineHeight
 } from '../design-tokens';
-import { heightNumber } from '../styles';
+import { DirectionalStyleSheetBehavior } from '../styles';
 
 /**
+ * LTR styles for calendar
  * @internal
  */
+const ltrStyles = css`
+.selected + .selected {
```

Comment:
I think this can all be done with the `inline` styles. `border-start-start-radius`, `border-end-start-radius`, `border-start-width`, `padding-inline-start`, and `margin-inline-start` and then we don't need the directional switch.

Reply:
This is really cool. I hadn't used the inline styles before. Updated and it looks good in both RTL and LTR.